addons/io/org.openhab.io.imperihome/README.md

```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
 
 The ImperiHome integration service can be installed through the Paper UI. Navigate to Extensions &gt; Misc and click Install.
```

Comment:
Just noticed: "Extensions" is by now called "Add-ons", so you might want to update this.